exercises/practice/twelve-days/twelve_days.cpp

```diff
@@ -0,0 +1,22 @@
+#include "twelve_days.h"
+#include <vector>
+#include <string>
+
+namespace twelve_days {
+
+std::string verse(int day) {
```

Comment:
This track does not provide stubs in general (except for some very early concept exercises). Remove those please.